### ISSUE TITLE
Line numbers match color of first character of line

### ISSUE THREAD

**Issue by beojan** [greduan/emacs-theme-gruvbox#89]
![image](https://user-images.githubusercontent.com/3727925/28678616-21e6dc46-72f1-11e7-8d08-72e195aa251b.png)

The image should make it clear.

**Comment by greduan** [comment]
Thanks for the bug report. What version of Emacs are you using? I ask because it may be relevant if you're building your own Emacs. :)

**Comment by beojan** [comment]
25.2

On 27 July 2017, at 17:45, Eduardo Lavaque <notifications@github.com> wrote:

Thanks for the bug report. What version of Emacs are you using? I ask because it may be relevant if you're building your own Emacs. :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.￼

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/greduan/emacs-theme-gruvbox","title":"greduan/emacs-theme-gruvbox","subtitle":"GitHub repository","main_image_url":"<a href="https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png">https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png</a>","avatar_image_url":"<a href="https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png">https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png</a>","action":{"name":"Open in GitHub","url":"<a href="https://github.com/greduan/emacs-theme-gruvbox">https://github.com/greduan/emacs-theme-gruvbox</a>"}},"updates":{"snippets":[{"icon":"PERSON","message":"@greduan in #89: Thanks for the bug report. What version of Emacs are you using? I ask because it may be relevant if you're building your own Emacs. :)"}],"action":{"name":"View Issue","url":"<a href="https://github.com/greduan/emacs-theme-gruvbox/issues/89#issuecomment-318402196">https://github.com/greduan/emacs-theme-gruvbox/issues/89#issuecomment-318402196</a>"}}}

**Comment by martijnat** [comment]
I just tried it out and I can reproduce this issue with linum-mode in emacs25, but not in emacs26 or emacs24.

The linum faces inherit from the native line numbers introduced in emacs26.  When loading the theme under emacs25 the linum faces inherit from an undefined face, which causes some weird fallback behaviour.

I will have a closer look at it later today and send a pull request once I have fixed it.
